.github/workflows/system-tests.yml

```diff
@@ -188,12 +192,12 @@ jobs:
         run: ./run.sh PARAMETRIC
 
       - name: Compress artifact
-        if: needs.needs-run.outputs.outcome == 'success' || github.event_name == 'schedule'
+        if: always() || github.event_name == 'schedule'
```

Comment:
@wconti27 Before I put a PR up that reverts this change, since it is causing https://github.com/DataDog/dd-trace-py/actions/runs/8214058742/job/22466141931?pr=8659, may I ask why this was changed in the first place?

Reply:
This was changed so that when a test fails, we still compress and upload an artifact, which wasn't happening before (we were skipping this step if the system tests run fails). I have this PR that fixes it: https://github.com/DataDog/dd-trace-py/pull/8658